analyses/SC_cd_2010/03_sim_SC_cd_2010.R

```diff
@@ -0,0 +1,63 @@
+###############################################################################
+# Simulate plans for `SC_cd_2010`
+# © ALARM Project, June 2022
+###############################################################################
+
+# Run the simulation -----
+cli_process_start("Running simulations for {.pkg SC_cd_2010}")
+
+# Custom constraints
+constr_sc <- redist_constr(map) %>%
+    add_constr_splits(strength = 0.5, admin = county_muni) %>%
+    add_constr_grp_hinge(5, vap_black, vap, 0.4) %>%
+    add_constr_grp_hinge(-5, vap_black, vap, 0.3) %>%
+    add_constr_grp_inv_hinge(2, vap_black, vap, 0.6)
+
+# Sample
+set.seed(2010)
+plans <- redist_smc(map,
+    nsims = 3000,
+    runs = 2L,
+    ncores = 1,
+    compactness = 1,
+    pop_temper =  0.01,
+    counties = county,
+    constraints = constr_sc) %>%
+    match_numbers("cd_2010")
```

Comment:
Once the `summary(plans)` output looks good, we will be thinning this 6000 plans to 5000. You can copy from other 2020 code that has been merged in.